### DISCUSSION TITLE
Remove objects from docs

### DISCUSSION THREAD

**Discussion by stefanodvx** [mkdocs/mkdocs#3539] [Q&A]
![image](https://github.com/mkdocs/mkdocs/assets/69367859/5019fcf3-aee2-4a35-909b-43766a428dd7)
This is my current documentation using mkdocstings and mkdocs-material theme. I see some unwanted extra objects are added to docs, like the Bases of class, the source code and the init. How can I decide which objects keep in the page?

**Comment by pawamoy** [comment]
Have you tried to read mkdoctrings' documentation? There are options to decide whether to show base classes and source code indeed.

**Reply by markallasread** [reply]
Presuming resolved, but, linking just in case: Specifically, https://mkdocstrings.github.io/python/usage/#globallocal-options — `show_bases` for ex.
